store/sqlstore/group_store.go

```diff
@@ -851,7 +863,7 @@ func (s *SqlGroupStore) ChannelMembersToRemove() ([]*model.ChannelMember, *model
 					AND GroupMembers.DeleteAt = 0
 				GROUP BY
 					Channels.Id,
-					GroupMembers.UserId)`
+					GroupMembers.UserId)`, channelScope)
 
```

Comment:
I’m wondering if the newly added branching statements in this file merit converting the SQL statements to Squirrel.

Reply:
Ok, I'm going to change it.

Reply:
Done.

Reply:
For me, at least in this case, I find the original SQL statements approach (much) more readable (you can easily read the SQL code at once, and add comments if needed) when compared to Squirrel.

Reply:
> For me, at least in this case, I find the original SQL statements approach (much) more readable (you can easily read the SQL code at once, and add comments if needed) when compared to Squirrel.

Couldn't agree more. It was harder to write and is harder to read.